media_remover/src/test/scala/MediaNotRequiredMessageProcessorSpec.scala

```diff
@@ -1,13 +1,16 @@
+import MediaNotRequiredMessageProcessor.Action
 import akka.actor.ActorSystem
+import akka.http.javadsl.model.HttpMessage.DiscardedEntity
```

Comment:
Might be an erroneous import? Normally would expect to see `akka.http.scaladsl` not `javadsl`

Reply:
Ah, yes.